content/en/docs/tasks/administer-cluster/reconfigure-kubelet.md

```diff
@@ -238,92 +249,62 @@ $ kubectl get no ${NODE_NAME} -o json | jq '.status.config'
+If you see an error, you can search for it in the Kubelet's log for additional
+details.
 
-### Observe that the Kubelet is using the new configuration
+#### Make more changes
 
-Once more, retrieve the Node with `kubectl get node ${NODE_NAME} -o yaml`, and
-look for a `Node.Status.Config` that reports the new configuration as `assigned`
-and `active`, with no errors.
+Follow the workflow above to make more changes and push them again. Each
+time you change the ConfigMap's contents, it gets a new name.
 
-### Reset the Node to use its local default configuration
+#### Reset the Node to use its local default configuration
 
-Finally, if you wish to reset the Node to use the configuration it was
-provisioned with, simply edit the Node with `kubectl edit node ${NODE_NAME}` and
-remove the `Node.Spec.ConfigSource` field.
+To reset the Node to use the configuration it was provisioned with, edit the
+Node using `kubectl edit node ${NODE_NAME}` and remove the
+`Node.Spec.ConfigSource` field.
 
-### Observe that the Node is using its local default configuration
+#### Observe that the Node is using its local default configuration
 
-After removing this subfield, you should eventually observe that `Node.Status.Config`
-has become empty, as all config sources have been reset to `nil` (indicating the local
+After removing this subfield, `Node.Status.Config` eventually becomes
+empty, since all config sources have been reset to `nil`, which indicates that the local
 default config is `assigned`, `active`, and `lastKnownGood`), and no error is reported.
 
 {{% /capture %}}
 
 {{% capture discussion %}}
 ## Kubectl Patch Example
-As mentioned above, there are many ways to change a Node's configSource.
-Here is an example command that uses `kubectl patch`:
 
-```
+You can change a Node's configSource using several different mechanisms.
+This example uses `kubectl patch`:
+
+```bash
 kubectl patch node ${NODE_NAME} -p "{\"spec\":{\"configSource\":{\"configMap\":{\"name\":\"${CONFIG_MAP_NAME}\",\"namespace\":\"kube-system\",\"kubeletConfigKey\":\"kubelet\"}}}}"
 ```
 
 ## Understanding how the Kubelet checkpoints config
 
 When a new config is assigned to the Node, the Kubelet downloads and unpacks the
-config payload as a set of files on local disk. The Kubelet also records metadata
+config payload as a set of files on the local disk. The Kubelet also records metadata
 that locally tracks the assigned and last-known-good config sources, so that the
 Kubelet knows which config to use across restarts, even if the API server becomes 
 unavailable. After checkpointing a config and the relevant metadata, the Kubelet 
-will exit if the assigned config has changed. When the Kubelet is restarted by the
-babysitter process, it will read the new metadata, and use the new config.
+exits if it detects that the assigned config has changed. When the Kubelet is
+restarted by the OS-level service manager (such as `systemd`), it reads the new
```

Comment:
We can also update lines 50-51 to use the same "OS-level service manager" terminology. It currently says "...you must be running a process manager (like systemd)..."